buildroot-external/scripts/rauc.sh

```diff
@@ -2,6 +2,19 @@
 set -e
 
 
+function prepare_rauc_signing() {
+    local key="/build/key.pem"
+    local cert="/build/cert.pem"
+
+    if [ ! -f "${key}" ]; then
```

Comment:
Maybe we could add some check here, like `[ -z "$GITHUB_ACTIONS" ]` - otherwise it could lead to unexpected results for developers who fork this repository and expect they can use the CI builds without any changes. Or we can make sure the step for adding keys in GH workflow fails in case the variables are not defined.

Reply:
Yeah I prefer to solve this in GitHub actions.